hub/powertoys/run.md

```diff
@@ -240,6 +240,21 @@ Examples:
 - `) calendar week::3/27/2022` to show the calendar week for a date value.
 - `) unix epoch::3/27/2022 10:30:45 AM` to convert the given time and date value into a Unix epoch timestamp.
 
+#### Custom formats
+The plugin has a setting were you can define custom formats. The setting is a multiline text box were you have ta add one format per line.
+The syntax is `<format name>=<format pattern>` for using the local time and `<format name>=UTC:<format pattern>` for using the Universal Time Convention (UTC).
+Supported are all [standard patterns](https://learn.microsoft.com/dotnet/standard/base-types/custom-date-and-time-format-strings) and the following ones:
+- `DOW`: Number of the day in the week.
```

Comment:
One more thought... Longer bulleted lists sometimes work better as tables. Let me know what you think.

Reply:
This is a bullet list currently?!

Reply:
From the layout in this case I like to keep the list.

Reply:
OK, the [style guide](https://learn.microsoft.com/style-guide/scannable-content/lists) recommends "A list should have at least two items but no more than seven items.", but this isn't much longer than that.